### PR TITLE
Update ruleset: Fox News

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
We'd better rewrite the ruleset so that we can ensure it's working and make it easier to maintain.

The outcome would be something like [this](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#example-ruleset-after-style-guidelines-are-applied-with-test-urls). @injust are you still interested in improving this ruleset?

**Comment by injust** [comment]
@gloomy-ghost Tried my best at cleaning up the mess, could you take a look at my changes?

File: src/chrome/content/rules/Fox-News.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	<target host="global.fncstatic.com" />
+
+	<!-- foxbusiness.com -->
+	<target host="foxbusiness.com" />
+	<target host="www.foxbusiness.com" />
+	<target host="video.foxbusiness.com" />
+	<target host="beta.video.foxbusiness.com" />
+
+	<!-- foxnews.com -->
+	<target host="foxnews.com" />
```
[the remark this answers is not in the document]
Comment (injust) [reply src/chrome/content/rules/Fox-News.xml]: My bad, for some reason I thought it redirected to an HTTP version. Fixing.

**Comment by gloomy-ghost** [comment]
Thanks!

**Pull request merged**